settings/language-praat.cson

```diff
@@ -0,0 +1,7 @@
+'.source.praat':
+  'editor':
+    'autoIndentOnPaste': true
+    'commentStart': '; '
+    # TODO(Tomasito) Add indent patterns for non-closing indentation blocks like 'optionmenu' or 'choice'
```

Comment:
Can you please remove this comment so I can merge your commits without subsequent fix?